### DISCUSSION TITLE
Prisma generate step breaking in docker file on M1 chip

### DISCUSSION THREAD

**Discussion by FauziKam** [prisma/prisma#12148] [Q&A]
I am using M1 chip laptop. And I am working with Prisma with nestjs and docker . When I try to build my docker the Prisma generate step breaks and gives following error -

```
#29 0.739 > npx prisma generate
#29 0.739 
#29 4.096 Prisma schema loaded from prisma/schema.prisma
#29 4.337 Error: Get config: Error: Command failed with ENOENT: /usr/local/lib/node_modules/prisma/query-engine-linux-arm-openssl-1.1.x cli get-config --ignoreEnvVarErrors
#29 4.337 spawn /usr/local/lib/node_modules/prisma/query-engine-linux-arm-openssl-1.1.x ENOENT
#29 4.352 npm ERR! code ELIFECYCLE
#29 4.352 npm ERR! errno 1
```

I have installed openssl-1.1.1g binaries since I was getting an error for missing openssl binaries. I also made changes in Prisma.schema file to add binaryTargets = ["openssl-1.1.1g"]. 

Any help or suggestions would be helpful

**Comment by janpio** [comment]
Does `/usr/local/lib/node_modules/prisma/query-engine-linux-arm-openssl-1.1.x` exist?

**Reply by FauziKam** [reply]
I am not able to check inside the container since the container doesn't run and break.

**Reply by janpio** [reply]
Can you maybe try to see before running `npx prisma generate`?

**Reply by vladi-stevanovic** [reply]
👋 Hello @FauziKam  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
